rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -307,7 +307,7 @@ error_code cellCameraInit()
 
 	std::lock_guard lock(g_camera->mutex);
 
-	if (g_camera->init)
+	if (g_camera->init.exchange(1))
```

Comment:
this doesnt make any sense.
the init should be set set at the end and inside the null handler return scope